code/game/area/areas.dm

```diff
@@ -28,6 +28,10 @@
 		power_environ = 0
 	power_change()		// all machines set to current power level, also updates lighting icon
 
+	icon = 'icons/turf/areas.dmi'
+	icon_state = "white"
+	blend_mode = BLEND_MULTIPLY
```

Comment:
Why is this set here, not in the `/area` definition? Why does it make icon updates outside of `update_icon`?

Reply:
This is where they should be, per the reasons discussed on discord.